tidb-cloud/data-service-postman-integration.md

```diff
@@ -0,0 +1,89 @@
+    3. Copy the Data App URL from the TiDB Cloud dialog, and then paste the URL to Postman for the import.
+
+5. After you paste the URL, Postman imports the Data App automatically as a new [collection](https://learning.postman.com/docs/collections/collections-overview). The name of the collection is in the `TiDB Data Service - <Your App Name>` format.
+
+    In the collection, the deployed endpoints are grouped under the **Deployed** folder and the un-deployed endpoints are grouped under the **Draft** folder.
+
+## Step 2. Configure your Data App API key in Postman
+
+Before running the imported Data App in Postman, you need to configure the API key for the Data App in Postman as follows:
+
+1. In the left navigation menu of Postman, click `TiDB Data Service - <Your App Name>` to open a tab for it on the right side.
+2. Under the `TiDB Data Service - <Your App Name>` tab, click the **Variables** tab.
+3. In the variable table, enter the public key and private key for your Data App in the **Current value** column.
+4. In the upper-right corner of the `TiDB Data Service - <Your App Name>` tab, click **Save**.
+
+## Step 3. Run Data App in Postman
+
+To run your Data App in Postman, take the following steps:
+
+1. In the left navigation pane of Postman, expand the **Deployed** or **Draft** folder, and then click your endpoint name to open a tab for it on the right side.
+2. Under the `<Your Endpoint Name>` tab, you can call your endpoint as follows:
+
+    - For an endpoint without parameters, you can click **Send** to call it directly.
+    - For an endpoint with parameters, you need to fill in the parameter values first, and then click **Send**.
+
+        - For a `GET` or `DELETE` request, fill in the parameter values in the **Query Params** table.
+        - For a `POST` or `PUT` request, click the **Body** tab, and then fill in the parameter values as a JSON object. If **Batch Operation** is enabled for the endpoint in TiDB Cloud Data Service, fill in the parameter values as an array of JSON objects.
+
+3. Check the response in the lower pane.
+
+4. If you want to call the endpoint again with different parameter values, you can edit the parameter values accordingly, and then click **Send** again.
+
+To learn more about the Postman usage, see [Postman documentation](https://learning.postman.com/docs).
+
+## Deal with new changes in Data App
+
+After a Data App is imported to Postman, TiDB Cloud Data Service will not automatically synchronize new changes of the Data App to Postman.
+
+If you want any new changes to reflect in Postman, you have to [follow the import process](/tidb-cloud/data-service-postman-integration.md#step-1-import-your-data-app-to-postman) once again. Because the collection name is unique in a Postman workspace, you can either use the latest Data App to replace the previously imported one or import the latest Data App as a new collection.
```

Comment:
```suggestion
If you want any new changes to reflect in Postman, you have to [follow the import process](#step-1-import-your-data-app-to-postman) once again. Because the collection name is unique in a Postman workspace, you can either use the latest Data App to replace the previously imported one or import the latest Data App as a new collection.
```